app/Policies/ProjectCommentPolicy.php

```diff
@@ -0,0 +1,21 @@
+<?php
+namespace App\Policies;
+use App\User;
+use App\ProjectComment;
+use Log;
+use Illuminate\Auth\Access\HandlesAuthorization;
+class ProjectCommentPolicy
```

Comment:
I edited the policies name so it makes more sense where it is being used. I didn't make another custom request because I would be just reusing or using code I didn't need. I just need to check if the user is the owner of the comment. I think it looks cleaner and is easier to understand then having the check directly in the controller.